src/utils/navigation.js

```diff
@@ -43,6 +55,7 @@ const switchPage = async (page) => {
   if (pageButton) {
     pageButton.click()
     switchedPage = true
+    scriptCurrentPage = page
```

Comment:
This will not be set if the initial page equals the page the script wants to switch to (now it is `Build` only) because it won't be trying to switch to it. Either the `scriptCurrentPage = page` could be moved down, or we can pass parameters to `checkPage` instead (and allow it to check for a subpage as well which will be needed anyway). I'll do update for that.